waku/factory/waku_conf.nim

```diff
@@ -86,6 +86,7 @@ type WakuConf* {.requiresInit.} = ref object
   relay*: bool
   lightPush*: bool
   peerExchange*: bool
+  peerExchangeClient*: bool
```

Comment:
```suggestion
peerExchangeService*: bool
  peerExchange*: bool
```

Look few line below, we started to adopt the convention of marking the *service* one, I'd recommend to keep the same to make it easier to work on the codebase.

Reply:
Actually, I am not sure `peerExchangeClient` is a good name. Because it implies we are enabling peer exchange as client. But it's more than that: we are enabling peer exchange discovery and setting up automated routine to do it.

Hence, something like `peerExchangeDiscovery` may be better.